src/components/uast/Node.js

```diff
@@ -136,6 +139,8 @@ export class Node extends Component {
         label="Node"
         highlighted={node.highlighted}
         collapsed={!node.expanded}
+        hovered={hoveredId === node.id}
+        toggle={() => toggle(node.id)}
```

Comment:
Why do we need the `toggle` here?

Reply:
oh. sorry. It came from here: https://github.com/bblfsh/dashboard/pull/113
Let's merge 113 first, then I'll rebase and we will continue with this PR, ok?

Reply:
ok